### PR TITLE
Updating README to remove the optionals

### PR DESCRIPTION
**Pull request by CheezItMan**
Adjusting Optionals and feedback template and I added a link to the feedback template in the README.  

Feel free to adjust from here, I just had 5 minutes between 1-on-1s.

### CODE REVIEW

File: feedback.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-|  Feature 	|  Feedback 	&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|
-|---	|---	|
-|  **Baseline** |  |
-|  Readable code with consistent indentation. 	| 	|
-|  **Primary Requirements**  |  |
-|  Created Custom Solar System Class with initialize, add planet & list planets methods, without using **puts**.	|   	|
-|  Planet Class Created |  |
-|  Created a collection of Planet objects as an instance variable in SolarSystem. |   |   
+|  Feature 	|  Feedback
+|---	|---
```
Comment (droberts-sea) [inline feedback.md]: You can get rid of the leading pipe-space-space here as well

**Review by droberts-sea — APPROVED** [review]
Looks good. I think Dee has some outstanding changes on this project, so you might want to double check with her.

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -3,9 +3,9 @@ Congratulations! You're submitting your assignment.
...
-|  Question 	|  Answer 	|
+|  Question 	|  Answer 	
 |---	|---	|
-|  What was the purpose of the `initialize` method in your class?	|   	|
-|  Describe an instance variable you used and what you used it for.	|   	|
-|  Describe what the difference would be if your `SolarSystem` used an `Array` vs a `Hash`. 	|   	|
-|  Do you feel like you used consistent formatting throughout your code? 	|   	|
+|  What was the purpose of the `initialize` method in your class?	|   	
+|  Describe an instance variable you used and what you used it for.	|   	
+|  Describe what the difference would be if your `SolarSystem` used an `Array` vs a `Hash`. 	|
```
Comment (droberts-sea) [inline .github/PULL_REQUEST_TEMPLATE]: Sorry to nitpick, but can you get rid of this trailing whitespace as well?

I believe Atom has an option to do this automatically.

**Comment by CheezItMan** [comment]
Left 1 space after the "|" simply because I hate text squished against it.

**Pull request merged**